apps/api/src/profiles/profiles.service.ts

```diff
@@ -90,4 +91,16 @@ export class ProfilesService {
       message: 'User info was updated succesfully',
     });
   }
+
+  collegeName() {
+    const data = readFileSync('./src/profiles/data.json', { encoding: 'utf8' });
+    const parseData = JSON.parse(data);
+    const collegeJSON = { data: [] };
+    for (let key = 0; key < parseData.length; key += 1) {
```

Comment:
change this to map function js, that is remove using for loop and use js functions instead